modules/swagger-codegen/src/main/resources/php/ApiClientConfiguration.mustache

```diff
@@ -0,0 +1,257 @@
+<?php
+/**
+ *  Copyright 2015 SmartBear Software
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+namespace {{invokerPackage}};
+
+class ApiClientConfiguration {
```

Comment:
Would it be possible to keep the original name "Configuration"? The reason is to make it consistent with other languages (e.g. Python, C#, etc), which use the name "Configuration"